numpy/ma/core.py

```diff
@@ -3692,7 +3709,7 @@ def __add__(self, other):
     #
     def __radd__(self, other):
         "Add other to self, and return a new masked array."
-        return add(self, other)
+        return add(other, self)
```

Comment:
So addition may not be commutative. That is also the case with `__numpy_ufunc__`. Could you put an explanatory comment here so that some well meaning individual doesn't "fix" it in the future.